doc/encoding.rdoc

```diff
@@ -132,7 +132,232 @@ returns the \Encoding of the concatenated string, or +nil+ if incompatible:
   s1 = "\xa1\xa1".force_encoding('euc-jp')     # => "\x{A1A1}"
   Encoding.compatible?(s0, s1)                 # => nil
 
-==== \Encoding Options
+=== \String \Encoding
+
+A Ruby String object has an encoding that is an instance of class \Encoding.
+The encoding may be retrieved by method String#encoding.
+
+The default encoding for a string literal is the script encoding
+(see Encoding@Script+encoding):
+
+  's'.encoding # => #<Encoding:UTF-8>
+
+The default encoding for a string created with method String.new is ASCII-8BIT
+if no argument is given, UTF-8 otherwise:
```

Comment:
This isn't correct.  If a string is given as an argument, then the returned string has the same encoding.  If you pass a literal string, you generally get UTF-8, because that is the default script encoding.

Reply:
Done.